tests/topotests/lib/common_config.py

```diff
@@ -676,7 +676,7 @@ def generate_support_bundle():
     for rname, rnode in router_list.items():
         logger.info("Generating support bundle for {}".format(rname))
         rnode.run("mkdir -p /var/log/frr")
-        bundle_log = rnode.run("python2 /usr/lib/frr/generate_support_bundle.py")
+        bundle_log = rnode.run("python /usr/lib/frr/generate_support_bundle.py")
```

Comment:
What if there is no ``python`` anymore, but only ``python3`` like in debian 11? Maybe something like this would be more apropriate:
```
if sys.version_info.major < 3:
	bundle_log = rnode.run("python2 /usr/lib/frr/generate_support_bundle.py")
else:
	bundle_log = rnode.run("python3 /usr/lib/frr/generate_support_bundle.py")
```

Reply:
hmm, confused: you were furious about python2, so I'm trying to eliminate it. I think it's useful to just be able to say "python", and I'll bet many developers will too, but if that's not acceptable I'd prefer just forcing "python3" and ensure that works.

Reply:
If i run that script with your changes with python3 it gives me this:
```
Traceback (most recent call last):
  File "/root/frr-source/tools/generate_support_bundle.py", line 123, in <module>
    processConfFile(lines)
  File "/root/frr-source/tools/generate_support_bundle.py", line 91, in processConfFile
    for line in lines:
  File "/root/frr-source/tools/generate_support_bundle.py", line 22, in <genexpr>
    lines = filter(None, (line.rstrip() for line in supportBundleConfFile))
ValueError: I/O operation on closed file.
```

Reply:
And the problem here is, that we need a solution for both python2 and python3, otherwise it will not pass the CI run.

Reply:
If @davischw is correct that there is no `python` executable in Debian then he's pointing out a legitimate bug in your python3 conversion. I don't see a need to trade jabs here.

I'm also fine forcing python3 as all our supported platforms should have it available at this time. Per https://docs.python.org/3/using/unix.html#miscellaneous you should be able to hardcode the path rather than leaving it to shell interpretation, but if you prefer that, `/usr/bin/env python3` should do the trick.